py/torch_tensorrt/dynamo/runtime/_PythonTorchTensorRTModule.py

```diff
@@ -270,3 +325,23 @@ def get_layer_info(self) -> str:
             trt.LayerInformationFormat.JSON
         )
         return engine_json
+
+    def cudagraphs_validate_shapes(self, inputs: Sequence[torch.Tensor]) -> bool:
+        """
+        Validates the input shapes of the forward function
+        versus the version currently active for the
+        """
+        # Representation of input shapes to a given model
+        # Shapes are concatenated as so:
+        # x: (3, 4), y: (4, 5) --> Key: (3, 4)(4, 5)
+        new_shape_key = "".join(str(tuple(t.shape)) for t in inputs)
```

Comment:
Can we have no whitespace in the key, ensures that there is only one valid representation of a key